classes/patreon_options.php

```diff
@@ -227,40 +211,14 @@ function patreon_plugin_setup_page(){
                                         <tr valign="top">
 											<th scope="row">
 												<strong>Allow Admins/Editors to 'Login with Patreon' Button</strong>
+												<div class="patreon-options-info">If on, admins and editors will be able to login to your website login via Patreon. If off, only non admin/editor users will be able to login to your website via Patreon. Recommended: on</div>
 											</th>
 											<td>
 												<input type="checkbox" name="patreon-enable-allow-admins-login-with-patreon" value="1"<?php checked( get_option('patreon-enable-allow-admins-login-with-patreon', false) ); ?> />
 											</td>
                                         </tr>
                                         <?php } ?>
```

Comment:
All of this could use some clean up in terms of spacing

Reply:
In terms of indentation and opening/closing of php tags and embedded PHP, these seem to be compliant with what they have at WP standards:

https://make.wordpress.org/core/handbook/best-practices/coding-standards/html/#indentation

Checking the indentation of HTML and PHP i see that the indentation as it exists is correct. It indeed looks too much, however. The indentation becomes too much since it starts all the way from the class method and renders an entire settings page in one function.

If i have misunderstood, please point to the direction which we must improve this bit.